controllers/toolchainconfig/deliveryserviceconfig.go

```diff
@@ -23,3 +23,7 @@ func (d DeliveryServiceFactoryConfig) GetMailgunSenderEmail() string {
 func (d DeliveryServiceFactoryConfig) GetMailgunReplyToEmail() string {
 	return d.Notifications().MailgunReplyToEmail()
 }
+
+//func (d DeliveryServiceFactoryConfig) GetNotificationEnvironment() string {
```

Comment:
probably we can delete this one if not needed

Reply:
done in [fb7de08](https://github.com/codeready-toolchain/host-operator/pull/754/commits/fb7de08aadf0e35b6ffd34b707f24526645a3feb)